test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12898,8 +12898,8 @@ def isUp(self, max_attempts=None):
             try:
                 nodes = self.server.status(NODE, id=self.shortname)
                 if nodes:
-                    self.server.expect(NODE, {'state': 'free'},
-                                       id=self.shortname)
+                    attr = {'state': (MATCH_RE, 'free|provisioning')}
```

Comment:
just wondering, is it okay to use a mom when it is in state provisioning? We kind of use the isUp() to mean that the daemon is ready to start running jobs.